plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/DBPScriptObject.java

```diff
@@ -66,6 +66,8 @@ public interface DBPScriptObject extends DBPObject {
     String OPTION_INCLUDE_PARTITIONS = "ddl.includePartitions"; //$NON-NLS-1$
 
     String OPTION_USE_SPECIAL_NAME = "ddl.useSpecialName"; //$NON-NLS-1$
+    
+    String OPTION_CAST_PARAMS = "ddl.castParams"; //$NON-NLS-1$
```

Comment:
It is not DDL-specific option. It is generic SQL option so lets name it "sql.castParameter".

Reply:
Ok, agreed